packages/editor/src/components/post-locked-modal/style.scss

```diff
@@ -15,5 +11,5 @@
 .editor-post-locked-modal__avatar {
 	border-radius: $radius-block-ui;
 	margin-top: $grid-unit-20;
-	margin-right: $grid-unit-30;
+	min-width: initial !important;
```

Comment:
The `HStack` sets `min-width: 0` to all its children. We must override that so the image can take up the entire width.